beacon-chain/sync/subscriber_beacon_attestation.go

```diff
@@ -49,12 +49,16 @@ func (*Service) aggregatorSubnetIndices(currentSlot primitives.Slot) []uint64 {
 	return slice.SetUint64(commIds)
 }
 
-func (*Service) attesterSubnetIndices(currentSlot primitives.Slot) []uint64 {
+func attesterSubnetIndices(currentSlot primitives.Slot) map[uint64]bool {
 	endEpoch := slots.ToEpoch(currentSlot) + 1
 	endSlot := params.BeaconConfig().SlotsPerEpoch.Mul(uint64(endEpoch))
-	var commIds []uint64
+
+	subnets := make(map[uint64]bool)
```

Comment:
```suggestion
	subnets := make(map[uint64]bool, int(endSlot-currentSlot))
```

minor optimization since you know the expected capacity

Reply:
`int(endSlot-currentSlot+1)` right?

Reply:
Yes :)

Reply:
Fixed in https://github.com/OffchainLabs/prysm/pull/15471/commits/6ab6e4b2c29b26057a532028bdb09cc71eee6644.